flixel/system/FlxAssets.hx

```diff
@@ -28,7 +28,7 @@ class GraphicLogo extends BitmapData {}
 class GraphicVirtualInput extends BitmapData {}
 
 @:file("assets/images/ui/virtual-input.txt")
-class VirtualInputData extends ByteArray {}
+class VirtualInputData extends #if lime_legacy ByteArray #else ByteArrayData #end {}
```

Comment:
Okay, so this issue is caused by the recent ByteArray refactor in OpenFL. I do believe it's a breaking change -- but apparently there was a good reason for it.

In any case, ByteArray is now an abstract (over ByteArrayData) and so you can't extend it directly. I'm asking Joshua if there's a better way to handle this, but for now this is all I can think of. What version should we check against, or should we just wait until the next OpenFL release before making this particular change?